itou/utils/templatetags/buttons_form.py

```diff
@@ -0,0 +1,90 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.inclusion_tag("./utils/templatetags/buttons_form.html", takes_context=False)
+def itou_buttons_form(
+    primary_label="Suivant",
+    form_title=None,
+    secondary_url=None,
+    reset_submit=False,
+    reset_url=None,
+    matomo_category=None,
+    matomo_action=None,
+    matomo_name=None,
+    show_mandatory_fields_mention=True,
+):
+    """
+    Render buttons on forms.
+
+    **Tag name**::
+
+        itou_buttons_form
+
+    **Parameters**::
+
+        primary_label
+            The label for the primary button.
+            Default: "Suivant"
+
+        form_title
+            The form's label, used for aria-label on the form element.
+            Optional
+            Default: None
+
+        secondary_url
+            The url for the secondary button.
+            Optional
+            Default: None
+
+        reset_submit
+            If True, display a reset button, whether reset_url exists or not.
+            Optional
+            Default: False
+
+        reset_url
+            The url for the reset button.
+            Optional
+            Default: None
+
+        matomo_category & matomo_action & matomo_name
+            If set, the buttons will send a matomo event on click.
+            Optional
+            Default: None
+
+        show_mandatory_fields_mention
+            if True, show the mention "champs obligatoires" on the form.
+            Default: TCue
+
+
+    **Usage**::
+
+        {% itou_buttons_form  %}
+
+    **Example**::
+
+        {% itou_buttons_form reset_submit=True %}
+    """
+    if form_title:
+        primary_aria_label = "Passer à l'étape suivante du formulaire {}".format(form_title)
+        secondary_aria_label = "Retourner à l'étape précédente du formulaire {}".format(form_title)
```

Comment:
```suggestion
        primary_aria_label = f"Passer à l'étape suivante du formulaire {form_title}"
        secondary_aria_label = f"Retourner à l'étape précédente du formulaire {form_title}"
```

Reply:
bien vu. 
je n'ai pas retrouvé de formulaire avec in `title`, cette possibilité est retirée.